bin/cyhy-kevsync

```diff
@@ -49,19 +49,26 @@ def parse_json(db, json_stream):
 
     # Gather all the cveIDs from the JSON file
     all_cve_ids = set()
+    known_ransomware_use_ids = set()
```

Comment:
Is it viable to use a list of tuples or dictionaries for this? I'm really not a fan of searching a list to derive a boolean value for every processed KEV ID.

Reply:
That's fair - please see https://github.com/cisagov/cyhy-core/pull/80/commits/2903d3f750d50b45c3a49947a62e4430ccf6e2d1.